members/people.json

```diff
@@ -655186,6 +655188,14 @@
       "post_id": "uk.org.publicwhip/cons/10840", 
       "start_date": "2016-05-09", 
       "start_reason": "regional_election"
+    }, 
+    {
+      "id": "uk.org.publicwhip/member/80686", 
+      "on_behalf_of_id": "labour", 
+      "person_id": "uk.org.publicwhip/person/13956", 
+      "post_id": "uk.org.publicwhip/cons/10836", 
+      "start_date": "2019-06-15", 
+      "start_reason": "replacement"
```

Comment:
It looks like this file uses `replaced_in_region` for this happening.

Reply:
Cheers, thanks - I looked for a previous similar occurrence but couldn't find one, and not being Scottish myself, I wasn't entirely sure as to how the mechanism for replacement worked. :)